### PR TITLE
add cross references pointing to the CLI docs

### PR DESCRIPTION
**Pull request by atravitz**
## Description
Within the API docs, add links to the new(-ish) command line interface docs.

## Motivation and Context
I think our documentation could benefit from more cross linking and I didn't know about `signac rm --clear` so I'm starting here.

## Types of Changes

- [x] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [x] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac/blob/master/changelog.txt) and added all related issue and pull request numbers for future reference (if applicable). See example below.

Example for a changelog entry: `Fix issue with launching rockets to the moon (#101, #212).`

### CODE REVIEW

File: signac/contrib/job.py [hunk 1]
```diff
@@ -145,6 +145,8 @@ def __eq__(self, other):
     def workspace(self):
         """Each job is associated with a unique workspace directory.
 
+        See :ref:`signac-cli-job` ``--workspace`` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:148]:
```suggestion
        See :ref:`signac job -w <signac-cli-job>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 2]
```diff
@@ -269,6 +271,8 @@ def statepoint(self):
             ``sp_dict = job.statepoint()`` instead of ``sp = job.statepoint``.
             For more information, see : :class:`~signac.JSONDict`.
 
+        See :ref:`signac-cli-statepoint` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:274]:
```suggestion
        See :ref:`signac statepoint <signac-cli-statepoint>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 3]
```diff
@@ -309,6 +313,8 @@ def document(self):
             persistent JSON file, use :attr:`~Job.document` instead of :attr:`~Job.doc`.
             For more information, see :attr:`~Job.statepoint` or :class:`~signac.JSONDict`.
 
+        See :ref:`signac-cli-document` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:316]:
```suggestion
        See :ref:`signac document <signac-cli-document>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 4]
```diff
@@ -486,6 +492,8 @@ def init(self, force=False):
 
         Returns the calling job.
 
+        See :ref:`signac-cli-job` ``--init`` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:495]: Looks like we named init/create differently between the CLI and Python API.
```suggestion
        See :ref:`signac job -c <signac-cli-job>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 5]
```diff
@@ -512,6 +520,8 @@ def clear(self):
         This function will do nothing if the job was not previously
         initialized.
 
+        See :ref:`signac-cli-rm` [-c] for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:523]:
```suggestion
        See :ref:`signac rm -c <signac-cli-rm>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 6]
```diff
@@ -543,6 +553,8 @@ def remove(self):
         This function will do nothing if the workspace directory
         does not exist.
 
+        See :ref:`signac-cli-rm` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:556]:
```suggestion
        See :ref:`signac rm <signac-cli-rm>` for the command line equivalent.
```

File: signac/contrib/job.py [hunk 7]
```diff
@@ -565,6 +577,8 @@ def move(self, project):
         This function will attempt to move this instance of job from
         its original project to a different project.
 
+        See :ref:`signac-cli-move` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/job.py:580]:
```suggestion
        See :ref:`signac move <signac-cli-move>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 8]
```diff
@@ -772,6 +775,8 @@ def build_job_statepoint_index(self, exclude_const=False, index=None):
     def detect_schema(self, exclude_const=False, subset=None, index=None):
         """Detect the project's state point schema.
 
+        See :ref:`signac-cli-schema` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:778]:
```suggestion
        See :ref:`signac schema <signac-cli-schema>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 9]
```diff
@@ -894,6 +899,8 @@ def find_jobs(self, filter=None, doc_filter=None):
         state points, whereas the `doc_filter` argument compares against job
         document keys.
 
+        See :ref:`signac-cli-find` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:902]:
```suggestion
        See :ref:`signac find <signac-cli-find>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 10]
```diff
@@ -1275,6 +1282,8 @@ def create_linked_view(self, prefix=None, job_ids=None, index=None, path=None):
             major difference that view hierarchies are actually *updated*,
             meaning that invalid links are automatically removed.
 
+        See :ref:`signac-cli-view` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1285]:
```suggestion
        See :ref:`signac view <signac-cli-view>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 11]
```diff
@@ -1369,6 +1378,8 @@ def clone(self, job, copytree=syncutil.copytree):
 
         Create an identical copy of job within this project.
 
+        See :ref:`signac-cli-clone` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1381]:
```suggestion
        See :ref:`signac clone <signac-cli-clone>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 12]
```diff
@@ -1407,6 +1418,8 @@ def sync(self, other, strategy=None, exclude=None, doc_sync=None, selection=None
         If a job is already part of this project, try to synchronize the job
         using the optionally specified strategies.
 
+        See :ref:`signac-cli-sync` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1421]:
```suggestion
        See :ref:`signac sync <signac-cli-sync>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 13]
```diff
@@ -1493,8 +1506,10 @@ def export_to(self, target, path=None, copytree=None):
 
         See Also
         --------
-        :meth:`~signac.Project.import_from` : Previously exported or non-signac
-            data spaces can be imported with.
+        :meth:`~signac.Project.import_from` :
+            Previously exported or non-signac data spaces can be imported.
+
+        See :ref:`signac-cli-export` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1512]:
```suggestion
        See :ref:`signac export <signac-cli-export>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 14]
```diff
@@ -1552,6 +1567,8 @@ def import_from(self, origin=None, schema=None, sync=None, copytree=None):
         --------
         :meth:`~signac.Project.export_to` : Export the project data space.
 
+        See :ref:`signac-cli-import` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1570]:
```suggestion
        See :ref:`signac import <signac-cli-import>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 15]
```diff
@@ -1839,6 +1856,8 @@ def index(self, formats=None, depth=0,
         The job document if it exists, is always indexed, other
         files need to be specified with the formats argument.
 
+        See :ref:`signac-cli-project`  ``--index`` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:1859]:
```suggestion
        See :ref:`signac project -i <signac-cli-project>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 16]
```diff
@@ -1995,6 +2014,8 @@ def init_project(cls, name, root=None, workspace=None, make_dir=True):
         configuration would conflict with the provided initialization
         parameters.
 
+        See :ref:`signac-cli-init` for the command line equivalent.
```
Comment (bdice) [inline signac/contrib/project.py:2017]:
```suggestion
        See :ref:`signac init <signac-cli-init>` for the command line equivalent.
```

File: signac/diff.py [hunk 17]
```diff
@@ -15,6 +15,8 @@ def diff_jobs(*jobs):
     jobs' state points. The comparison is performed over the combined set of
     keys and values.
 
+    See :ref:`signac-cli-diff` for the command line equivalent.
```
Comment (bdice) [inline signac/diff.py:18]:
```suggestion
    See :ref:`signac diff <signac-cli-diff>` for the command line equivalent.
```

File: signac/contrib/project.py [hunk 18]
```diff
@@ -320,6 +320,9 @@ def workspace(self):
             The configuration will respect environment variables,
             such as ``$HOME``.
 
+
```
Comment (bdice) [inline signac/contrib/project.py:323]: too many \n
```suggestion
```

File: signac/contrib/project.py [hunk 19]
```diff
@@ -1493,8 +1505,10 @@ def export_to(self, target, path=None, copytree=None):
 
         See Also
         --------
-        :meth:`~signac.Project.import_from` : Previously exported or non-signac
-            data spaces can be imported with.
+        :meth:`~signac.Project.import_from` :
+            Previously exported or non-signac data spaces can be imported.
+
```
Comment (atravitz) [inline signac/contrib/project.py:1510]: I think we need another line here to keep it out of the `See Also` box

Comment (bdice) [reply signac/contrib/project.py:1510]: fixed!

**Review by bdice — APPROVED** [review]
I checked all the links, seems to render correctly! Thanks @atravitz.

**Comment by vyasr** [comment]
This is great, thanks @atravitz! Might I suggest also adding a table somewhere that lists all the correspondences (or at list the most relevant ones)?

**Comment by bdice** [comment]
> This is great, thanks @atravitz! Might I suggest also adding a table somewhere that lists all the correspondences (or at list the most relevant ones)?

I'd recommend merging this as-is and do that in a follow-up PR.

**Comment by atravitz** [comment]
> > This is great, thanks @atravitz! Might I suggest also adding a table somewhere that lists all the correspondences (or at list the most relevant ones)?
> 
> I'd recommend merging this as-is and do that in a follow-up PR.

Agreed

**Comment by vyasr** [comment]
> > > This is great, thanks @atravitz! Might I suggest also adding a table somewhere that lists all the correspondences (or at list the most relevant ones)?
> > 
> > 
> > I'd recommend merging this as-is and do that in a follow-up PR.
> 
> Agreed

That's fine with me, just make an issue so it's not forgotten.

**Review by pepak13 — APPROVED** [review]

**Comment by bdice** [comment]
Issue created: https://github.com/glotzerlab/signac/issues/412

**Pull request merged**